compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -4484,7 +4484,7 @@ declare_lint! {
     ///
     /// `refining_impl_trait` is a lint group composed of two lints:
     ///
-    /// * `refining_impl_trait_reachable`, for refinements that are publically
+    /// * `refining_impl_trait_reachable`, for refinements that are publicly
```

Comment:
doesn't need changing; likewise the one just below

Reply:
done.